SourceS/sdl_compat.h

```diff
@@ -1,6 +1,12 @@
 // Compatibility wrappers for SDL 1 & 2.
 #include <SDL.h>
 
+#if defined(__AMIGA__) // Add other systems that require an 8bit screen here
+#define D_BPP 8

```

Comment:
This will be easy to configure via CMake once the controller port is merged.
I'll need it in the future to improve RG350 port performance as well, as it has a BGR screen and the most optimal `D_BPP` value for it is `15` on SDL1 (`15` is just like `16` but BGR instead of RGB).

Reply:
We can now use `SDL1_VIDEO_MODE_BPP` (as of https://github.com/diasurgical/devilutionX/commit/2acb87c743b9b0ff7ed5285d0dc2b819903b10e2)